phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/LocalRepository.java

```diff
@@ -239,6 +279,15 @@ public RepositoryDTO fetchInstallableApplications() {
         return results;
     }
 
+    private TypeDTO unSerializeType(File jsonFile) {
```

Comment:
I would prefer `unserializeType`, this looks more natural than `unSerializeType`